docs/griptape-cloud/structures/index.md

```diff
@@ -0,0 +1,44 @@
+# Structures
+
+Use [Structures](https://cloud.griptape.ai/structures) to host any python code in the cloud in seconds.
+
+## Create a Structure
+
+1. First [Connect Your GitHub Account](https://cloud.griptape.ai/account)
+1. Install the [Griptape Cloud GitHub app](https://github.com/apps/griptape-cloud/installations/new/) to any account or organization you'd like to pull code in from
+    1. Be sure to allow the app access to `All Repositories` or select the specific repositories you need
```

Comment:
Missing periods.

Reply:
Eh, it's usually standard in lists to not end items with periods. I can add them if we think it reads better.